File: database/src/main/java/bao/webapp/repository/OrderRepository.java
package bao.webapp.repository;

import java.util.List;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import bao.webapp.model.Order;

@Repository
public interface OrderRepository extends CrudRepository<Order, Long> {

	List<Order> findByUserID(long userID);
}

File: database/src/main/kotlin/com/ossp/database/repository/OrderRepository.kt
package com.ossp.database.repository

import com.ossp.database.model.Order
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface OrderRepository : CrudRepository<Order, Long> {

    fun findByUserID(userID: Long?): List<Order>?
}
